qcodes/instrument/parameter.py

```diff
@@ -316,6 +322,38 @@ def __init__(self, name: str,
         # to perform a validation on get
         self._validate_on_get = False
 
+    @property
+    def raw_value(self) -> ParamRawDataType:
+        """
+        Represents the cached raw value of the parameter.
+
+        Setting the ``raw_value`` is not recommended as it may lead to
+        inconsistence state of the parameter.
+
+        Note that this property will be deprecated soon.
+        """
+        return self._get_cache_raw()
+
+    @raw_value.setter
+    def raw_value(self, new_raw_value: ParamRawDataType) -> None:
+        self._set_cache_raw(new_raw_value)
+
+    def _set_cache_raw(self, raw_value: ParamRawDataType) -> None:
```

Comment:
Do we really need these 2 private methods ?

Reply:
Basically, having these methods allows to limit the number of the references to `._latest`. 

`_get_cache_raw` is used internally in Parameter/_BaseParameter, and will be superceeded later in another PR by a public `get_cache_raw`.

`_set_cache_raw` is used only in the `raw_value` setter, and serves only the purpose of limiting references to `._latest`.

Would you agree with this approach?

Reply:
I don't feel strongly about it but private setters feels pointless to me. IMHO setters and getters are needed to avoid leaking the (private) storage implementation of the date but since the method is private too there is no worry about leaking. 

They may also be used to do some convert or cast but that is not the case here.

Reply:
I see. Let me leave it for now. at the moment i see the value of explicitly pointing that the new "cache stuff" is layered between "raw_value" and the "_latest". Can always be removed in later PRs during "further refactoring"